content/espanol/sql_workshop/activity12/_activity12.js

```diff
@@ -0,0 +1,60 @@
+function sql() 
+{
+  clearTable();
+  document.getElementById("sqlcommand").innerHTML = "ERROR INVALID INPUT";
+  document.getElementById("sqlcommand").style.visibility="visible";
+
+  var user = document.getElementById("commands").innerHTML;
+  debug(user.toLowerCase(),"select * from people;");
+  var array = alasql("MATRIX OF " + user);
+  var ans = JSON.stringify(alasql(user));
+
+  document.getElementById("sqlcommand").style.visibility = 'hidden';
+
+  ans = ans.substring(ans.indexOf("{") + 1, ans.indexOf("}"));
+  
+  var ansArr = ans.split(',');
+  var headArr = [];
+  var i;
+  for( i = 0; i < ansArr.length; i++)
+  {
+    headArr.push(ansArr[i].substring( ansArr[i].indexOf('"') + 1, ansArr[i].indexOf('":')));
+  }
+
+  sqlToTable(headArr, array, "table");
+  
+  /*change answer here */
+  var ans1 = "select * from people;".replace(/\s+/g, '');
+  
+  // Change string to lower case
+  var input = user.toLowerCase().replace(/\s+/g, '');
+  
+  if(input == ans1)
+  {
+    document.getElementById("story").innerHTML = "That's the correct command to show all the attendees!";
+    
+    /* Changes terminal screen */
+    var screen = document.getElementById("terminal_div");
+    screen.style.backgroundImage = "url('../media/green.png')"
+  }
+}
+
+function check()
+{
+   let potential = document.getElementById("person").innerHTML.trim().replace(/&nbsp;/g, '');
+   let trimmedPotential = potential.toLowerCase().replace(/\s+/g,'');
+
+   if (trimmedPotential == "dr.phi")
+   {
+    document.getElementById("answer").innerHTML = "You found the inventor! Now he can combine the Legendary Totems of Fun into a more powerful Totem! This Totem unleashes a force field around the Planet of Fun! The planet is saved from the invaders!";
```

Comment:
Translate text to the right of all instances of `.innerHTML = `

Reply:
Fixed, please review again